polygon-digital-carbon/src/templates/ICRCarbonContractRegistry.ts

```diff
@@ -18,6 +18,7 @@ export function handleNewICC(event: ProjectCreated): void {
   for (let i = 0; i < ICR_PROJECT_INFO.length; i++) {
     if (event.params.projectAddress.toHexString().toLowerCase() == ICR_PROJECT_INFO[i][0]) {
       project.id = ICR_PROJECT_INFO[i][1]
+      project.projectID = ICR_PROJECT_INFO[i][1]
```

Comment:
These seem redundant, no?

Reply:
`projectID` is used throughout the codebase already expecting a `registry-registryNum` format so instead of adjusting throughout, imo it's better if we edit at the source to bring in line with the Verra/everything else format